.github/workflows/ee_cli_release_fips.yml

```
@@ -0,0 +1,44 @@
+      - name: Publish linux-x64 exec to github
+        id: build-and-release-binary
+        uses: wangyoucao577/go-release-action@8fa1e8368c8465264d64e0198208e10f71474c87  # v1.50
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          goos: linux
+          goarch: amd64
+          goversion: 1.24.0
+          project_path: ./ee/cli/cmd/digger
+          binary_name: digger
+          pre_command: export CGO_ENABLED=0
+          ldflags: ${{ matrix.ldflags }}
```

actionlint (1.7.7)

property "ldflags" is not defined in object type {}

(expression)